app/views/layouts/_taskbar.html.haml

```diff
@@ -3,8 +3,11 @@
   %div{:style => "float: left;"}
     - defered_toolbar_render('summary_center_tb', controller.send(:restful?) ? "summary_center_restful_tb" : "summary_center_tb")
 
-= render :partial => "layouts/center_buttons"
-= render :partial => "layouts/x_custom_buttons"
+- defered_toolbar_render('center_tb', center_toolbar_filename)
+
+- if fname = custom_toolbar_filename
```

Comment:
Do we need the condition here? Seems like we'd have to put it everywhere for `custom_tb` so maybe it would make more sense to just do `return unless toolbar_name` in `defered_toolbar_render`? (It should pretty much already support it, but maybe a nil value can mean something else there?)